.env.development

```diff
@@ -23,9 +23,12 @@ VITE_TRSRY_ADDR=7L53bUTBopuwFt3mKUfmkzgGLayYa1Yvn1hAg9v5UMrQzTfh
 VITE_WC_PROJECT_ID=c47a5369367ec2dad6b49c478eb772f9
 VITE_HDX_CAIP_ID=polkadot:afdc188f45c71dacbaa0b62e16a91f72
 VITE_FF_WALLET_CONNECT=true
-VITE_STABLECOIN_ASSET_ID=15
+VITE_STABLECOIN_ASSET_ID=2
 VITE_STABLECOIN_ASSET_SYMBOL=tether
 VITE_FF_SETTINGS_ENABLED=true
 VITE_FF_BONDS_ENABLED=true
 VITE_REFERENDUM_LINK=https://hydradx.subsquare.io/democracy/referendum
 VITE_REFERENDUM_DATA_URL=https://hydradx.subsquare.io/api/democracy/referendums
+VITE_EVM_CHAIN_ID=222222
```

Comment:
you should be able to load this from rpc `eth.getChainId()`

Reply:
I don't think I can do this before I add the chain to the wallet, so I need it defined in advance.